src/test/ui/issue-22644.stderr

```diff
@@ -2,66 +2,46 @@ error: `<` is interpreted as a start of generic arguments for `usize`, not a com
   --> $DIR/issue-22644.rs:16:31
    |
 16 |     println!("{}", a as usize < long_name);
-   |                               ^ --------- interpreted as generic arguments
-   |                               |
-   |                               not interpreted as comparison
-   |
-help: if you want to compare the casted value then write:
-   |
-16 |     println!("{}", (a as usize) < long_name);
-   |                    ^^^^^^^^^^^^
+   |                    ---------- ^ --------- interpreted as generic arguments
+   |                    |          |
+   |                    |          not interpreted as comparison
+   |                    help: try comparing the casted value: `(a as usize)`
```

Comment:
Huh, changing the message string somehow changes formatting? Is it length dependent?